revolv/payments/tests/models/test_payment.py

```diff
@@ -0,0 +1,112 @@
+from django.core.management import call_command
+from django.db.models.signals import post_save
+from django.test import TestCase
+from django_facebook.utils import get_user_model
+from revolv.base.models import RevolvUserProfile
+from revolv.base.signals import create_profile_of_user
+from revolv.payments.models import Payment, PaymentInstrumentType
+from revolv.project.models import Project
+
+
+class PaymentTest(TestCase):
+
+    def setUp(self):
```

Comment:
We could do like `self.reinvestment = PaymentInstrumentType.objects.get_reinvestment()` to save a little verbosity here: we're doing `PaymentInstrumentType.objects.get_<something>()` a lot in these tests haha